npe2/_plugin_manager.py

```diff
@@ -220,6 +221,7 @@ def __init__(
         self._contrib = _ContributionsIndex()
         self._manifests: Dict[PluginName, PluginManifest] = {}
         self.events = PluginManagerEvents(self)
+        self._shims: List[NPE1Shim] = []
```

Comment:
Maybe rename this field to `_npe1_shims`.  Similarly, `test_shims.py` -> `test_npe1_shims.py`. I feel like being a bit pedantic with names will keep it clear what this was for later on. I think this and the test are the only spots where I felt a rename would be nice. The rest is clear.

Reply:
renamed everything to `NPE1Adapter`  `_npe1_adapters`, etc...